__tests__/app.test.js

```diff
@@ -105,7 +106,42 @@ describe("GET", () => {
           });
         });
     });
+    test("STATUS 200, should return the articles filtered by the topic value specified in the query", () => {
+      return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        expect(articles.length).not.toBe(0)
+        articles.forEach((article) => {
+          expect(typeof article).toBe('object');
+          expect(article['topic']).toBe("mitch");
+        })
+      })
+    })
+    test("STATUS 200, should return the articles if query is omitted", () => {
+      return request(app)
+      .get("/api/articles?topic")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        expect(articles.length).not.toBe(0)
+        articles.forEach((article) => {
+          expect(typeof article).toBe('object');
+        })
+      })
+    })
+    test("STATUS 404, should return an appropriate messasge if article does not exist with the matched query", () => {
+      return request(app)
+      .get("/api/articles?topic=ragini")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe('article does not exist for matched query');
+      })
+    })
+
```

Comment:
Missing a test here. What if the topic does exist but has no articles? For example `topic=paper`?